src/richie/apps/courses/templates/courses/cms/organization_fragment.html

```diff
@@ -0,0 +1,16 @@
+{% if current_page.publisher_is_draft %}
+    {% if organization.check_publication is True %}
+        <li class="course-detail__content__organizations__item{% if is_main %}--main{% endif %}">
+            {{ organization.public_extension.extended_object.get_title }}
+        </li>
+    {% else %}
+        <li class="course-detail__content__organizations__item--draft{% if is_main %} course-detail__content__organizations__item--main{% endif %}">
```

Comment:
@sveetch ok for your above comment but why don't we write this here then?
```
<li  class="course-detail__content__organizations__item course-detail__content__organizations__item--draft{% if is_main %} course-detail__content__organizations__item--main{% endif %}">
```

Reply:
Because i missed it :) You are correct this is the right way to do

Reply:
Ok so I need to change all templates as well: subject, organization, course...